clusterman/autoscaler/autoscaler.py

```diff
@@ -71,6 +72,7 @@ def __init__(
         self.cluster = cluster
         self.pool = pool
         self.scheduler = scheduler
+        logger.info(self.scheduler)
```

Comment:
Is this here for testing? Just logging `mesos` or `kubernetes` here is a little weird.

Reply:
whoops, yea, just testing.